opensrp-app/src/main/java/org/smartregister/util/DatabaseMigrationUtils.java

```diff
@@ -136,4 +140,56 @@ public static void recreateSyncTableWithExistingColumnsOnly(SQLiteDatabase datab
         database.endTransaction();
 
     }
+
+    public static void createAddedECTables(SQLiteDatabase database, List<String> bindings, CommonFtsObject commonFtsObject) {
+        ArrayList<CommonRepositoryInformationHolder> bindTypes = org.smartregister.Context.bindtypes;
+        for (CommonRepositoryInformationHolder bindType : bindTypes) {
+            if (bindings.contains(bindType.getBindtypename())) {
```

Comment:
The `List` contains method is a bit inefficient (especially for long lists), is it possible to replace `bindings` type to set?